tests/gramex.yaml

```diff
@@ -1279,6 +1279,14 @@ url:
         class: NER
       xsrf_cookies: false
 
+  mlpredictor:
+    pattern: /predictor
+    handler: MLPredictor
+    kwargs:
+      model: $YAMLPATH/predictor.pkl
+      data:
```

Comment:
Like FormHandler, allow `url:` to be directly under kwargs since we don't (yet) support multiple datasets. (And even if we do, we can handle it just like FormHandler)

Similarly, use $YAMLPATH/predictor/model.pkl